composer.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "keboola/http-extractor",
+    "license": "MIT",
+    "type": "project",
+    "description": "Keboola HTTP Extractor",
+    "keywords": ["keboola", "http", "extractor"],
+    "authors": [
+        {
+            "name": "Tomáš Fejfar",
+            "email": "tomas.fejfar@keboola.com"
+        }
+    ],
+    "autoload": {
+        "psr-4": {
+            "Keboola\\": "src/Keboola/"
+        }
+    },
+    "require-dev": {
+        "phpunit/phpunit": "~6.5",
+        "squizlabs/php_codesniffer": "~3.2",
+        "phpstan/phpstan-shim": "^0.9.2",
+        "jakub-onderka/php-parallel-lint": "^0.9.2"
+    },
+    "scripts": {
+        "tests": "phpunit",
+        "phpstan": "phpstan analyse ./src --level=max --no-progress",
+        "phpcs": "phpcs --standard=psr2 -n --ignore=vendor --extensions=php .",
+        "phplint": "parallel-lint -j 10 --exclude vendor .",
+        "build": [
+            "@phplint",
+            "@phpcs",
+            "@phpstan",
+            "@tests"
+        ],
+        "ci": [
+            "@composer validate --no-check-all",
```

Comment:
`--strict` (viz https://blog.martinhujer.cz/17-tips-for-using-composer-efficiently/)